docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -218,7 +218,17 @@ To improve the Developer Experience when developing or using the administration
 
 ### Exceptions
 
-- `uid` — This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+**`uid`**
+
+This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+
+The field type also allows an `options` key. The value is an `object` of options passed to the `uid` generator.
+
+- `separator` (string) — Specify the string to be used for word separation. Allowed separators are: `"-"`, `"_"`, `"."`, and `"~"`. Default: `"-"`
```

Comment:
Can you add a link to reference back to the slugify pacakge we use :)

Reply:
I've added a link to the package